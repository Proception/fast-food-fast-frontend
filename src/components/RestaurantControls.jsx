import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';

export default class RestaurantControls extends Component {
  render() {
    return (
      <section class="order">
          <div class="container">
            <div class="controls">
              <div class="search-box">
                <input type="text" name="" placeholder="Search Food Items"></input>
              </div>
              <div class="search-button">
                <input class="button" type="button" name="" value="Search"></input>
              </div>
              <div class="sort-controls">
                <select>
                    <option value="0" label="Sort by:"></option>
                    <option value="1" label="Available"></option>
                    <option value="2" label="Finished"></option>
                </select>
              </div>
            </div>
          </div>
        </section>
    )
  }
}
