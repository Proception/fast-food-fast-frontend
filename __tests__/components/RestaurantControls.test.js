import React from 'react';
import { shallow } from 'enzyme';
import RestaurantControls from '../../src/components/RestaurantControls.jsx';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<RestaurantControls/>);
});
describe('RestaurantControls Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
