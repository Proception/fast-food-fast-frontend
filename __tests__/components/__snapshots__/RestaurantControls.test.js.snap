// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestaurantControls Component should match snapshot 1`] = `
<section
  className="order"
>
  <div
    className="container"
  >
    <div
      className="controls"
    >
      <div
        className="search-box"
      >
        <input
          name=""
          placeholder="Search Food Items"
          type="text"
        />
      </div>
      <div
        className="search-button"
      >
        <input
          className="button"
          name=""
          type="button"
          value="Search"
        />
      </div>
      <div
        className="sort-controls"
      >
        <select>
          <option
            label="Sort by:"
            value="0"
          />
          <option
            label="Available"
            value="1"
          />
          <option
            label="Finished"
            value="2"
          />
        </select>
      </div>
    </div>
  </div>
</section>
`;
